README.md

```diff
@@ -1,163 +1,160 @@
+- [utils](/packages/utils/README.md)
+- [weather-api](/packages/weather-api/README.md)
 
 ## Getting started
 
-### Secrets
+Comprehensive setup instructions are available in the [Tupaia dev onboarding](https://beyond-essential.slab.com/posts/tupaia-system-architecture-bx4yroqt) series.
 
-Most packages will require a .env file. `.env.example` files indicate the required variables per package.
+### Secrets
 
-🔑 **BES internal:** [Adding .env files](https://beyond-essential.slab.com/posts/tupaia-monorepo-setup-v5egpdpq#step-3-add-env-files)
+Most packages will require a `.env` file. `.env.example` files indicate the required variables per package. More instructions for setting environment variables are in the [Tupaia monorepo setup](https://beyond-essential.slab.com/posts/tupaia-monorepo-setup-v5egpdpq#hvfnz-set-environment-variables) documentation.
 
-### Local database
+### Development database
 
-🔑 **BES internal:** [Tupaia monorepo setup](https://beyond-essential.slab.com/posts/tupaia-monorepo-setup-v5egpdpq) - steps 4 and 5
+Development database setup instructions are in the [Tupaia monorepo setup](https://beyond-essential.slab.com/posts/tupaia-monorepo-setup-v5egpdpq#hs8ne-set-up-database) documentation.
 
 ### Dependencies
 
-We use yarn workspaces to manage our packages, which allows us to run `yarn` once at the project
-root, and it will install dependencies everywhere.
+We use yarn workspaces to manage our packages, which allows us to run `yarn` once at the project root, and it will install dependencies everywhere.
 
 ## CI/CD
 
-We use codeship for the CI/CD
-
-🔑 **BES internal:** [CI/CD using Codeship](https://beyond-essential.slab.com/posts/ci-cd-using-codeship-uzxspw8z)
+We use [GitHub Actions](https://docs.github.com/en/actions) for CI/CD.
 
 ## Tests
 
 Most of the packages support the following scripts for testing:
 
-```
-
+```sh
 yarn test
-yarn test:coverage # also displays code coverage
-
+yarn test:coverage  # Also displays code coverage
 ```
 
-## Style Guide
+This project is also tested with [BrowserStack](https://www.browserstack.com).
+
+## Style guide
 
 We use a combination of [ESlint configs](https://eslint.org/docs/user-guide/configuring) to detect quality and formatting issues in code:
 
 - [@beyondessential/eslint-config-js](https://www.npmjs.com/package/@beyondessential/eslint-config-js) for JavaScript packages
 - [@beyondessential/eslint-config-ts](https://www.npmjs.com/package/@beyondessential/eslint-config-ts) for TypeScript packages
-- [@beyondessential/eslint-config-jest](https://www.npmjs.com/package/@beyondessential/eslint-config-jest) for packages using `Jest`
+- [@beyondessential/eslint-config-jest](https://www.npmjs.com/package/@beyondessential/eslint-config-jest) for packages using [Jest](https://jestjs.io)
 
 The config for this repository is defined in `.eslintrc` under the root folder. Additional rules/overrides per package are specified in this file.
```

Comment:
Is this outdated? We only seem to have `.eslintrc-*.json` files, but I’m not familiar so didn’t touch this